tests/test_batch.py

```diff
@@ -101,3 +102,18 @@ def test_batch_declare_complete_and_wait_for_results(self, request_mock):
         )
         assert batch.jobs[self.job_id].batch_id == batch.id
         assert batch.jobs[self.job_id].result == self.job_result
+
+    @pytest.mark.parametrize("device_type, configuration", [
+        (DeviceType.EMU_SV, {"precision": "normal"}),
+        (DeviceType.EMU_SV, Configuration()),
+    ])
+    def test_create_batch_configuration(self, device_type, configuration):
+        job = {"runs": self.n_job_runs, "variables": self.job_variables}
+        configuration = {"precision": "normal"}
```

Comment:
In this case, shouldn't we make use of the dataclass and have: configuration=Configuration(precision="normal")?

Reply:
Actually this line shouldn't be here at all, not sure where it came from. 
It overrides the parameterized test... (where I wanted to use a dict just to have a test using both Configuration and Dict types)